src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -255,20 +255,20 @@ export class OsmosisSwapper implements Swapper<ChainId> {
       const getFeeDataInput: Partial<GetFeeDataInput<OsmosisSupportedChainId>> = {}
       const ibcFromCosmosFeeData = await cosmosAdapter.getFeeData(getFeeDataInput)
 
-      const { tradeId } = await performIbcTransfer(
-        transfer,
-        cosmosAdapter,
+      const { tradeId } = await performIbcTransfer({
+        input: transfer,
+        adapter: cosmosAdapter,
         wallet,
-        osmoUrl,
-        'uatom',
-        COSMO_OSMO_CHANNEL,
-        ibcFromCosmosFeeData.fast.txFee,
+        blockBaseUrl: osmoUrl,
+        denom: 'uatom',
+        sourceChannel: COSMO_OSMO_CHANNEL,
+        feeAmount: ibcFromCosmosFeeData.fast.txFee,
         accountNumber,
         ibcAccountNumber,
         sequence,
-        ibcFromCosmosFeeData.fast.chainSpecific.gasLimit,
-        'uatom',
-      )
+        gas: ibcFromCosmosFeeData.fast.chainSpecific.gasLimit,
+        feeDenom: 'uatom',
+      })
```

Comment:
Same implementation, simply LSP converted to destructured args for quality of life